src/System.Net.Security/src/System/Net/Security/SecureChannel.cs

```diff
@@ -997,28 +867,16 @@ internal void ProcessHandshakeSuccess()
                     _trailerSize = streamSizes.Trailer;
                     _maxDataSize = checked(streamSizes.MaximumMessage - (_headerSize + _trailerSize));
                 }
-                catch (Exception e)
+                catch (Exception e) when (!ExceptionCheck.IsFatal(e))
```

Comment:
> since we've added a try/catch

I didn't add a try/catch.  Are you saying we should think about removing the try/catch?  We could, but I one of my goals for this pass was to not remove any tracing... the only tracing I removed was because it was obviously duplicative (or in a few cases where it was in a TRACE_VERBOSE region, had gotten stale, and referred to things that no longer existed).